Movie page

ANschoolacc opened the pull request:
The database is now populated with movies.
Made a component to render a single movie with title, poster, imdb rating, category and description when visiting /:id. The id in the path is the randomly generated id from mongoDB. 
Also made error handler page if an error occurs when visiting /:id so the server doesn't crash.

MickeMarten commented on kino-nextjs/app/lib/data.ts:
```diff
@@ -1,5 +1,8 @@
 import { moviesList } from "../../public/mockMovies";
```
This can be deleted? Or are you using the mockData somewhere?

ANschoolacc replied:
Yes it can. I just left it because i made a example in my last PR of rendering movies on all movies page.

MickeMarten commented on kino-nextjs/app/(movies)/[id]/error.tsx:
```diff
@@ -0,0 +1,25 @@
+    error,
+    reset,
+}: {
+    error: Error & { digest?: string };
```
I'm gonna Google digest.

MickeMarten reviewed (approved):
Really Nice work with the database. 
I'm thinking about if the database should be connected all the time. When we start doing the post and delete don't we need it to be connected all the time?

ConnyPettersson reviewed (approved):
Looks fine!

oskber reviewed (approved):
Good job! 
Also, good thoughts by Micke about the database connection.

ANschoolacc commented:
After your comments i looked up the mongoose docs on connections and you where right. It is better to open up the connection on app startup and keep it open, so i changed that in the last commit and moved the connection outside of function.
